packages/@aws-cdk/aws-location-alpha/lib/place-index.ts

```diff
@@ -198,11 +199,4 @@ export class PlaceIndex extends PlaceIndexBase {
     this.placeIndexUpdateTime = placeIndex.attrUpdateTime;
   }
 
-  private generateUniqueId(): string {
```

Comment:
This method can be generally used for creating resource names in location, so it has been extracted into a util.